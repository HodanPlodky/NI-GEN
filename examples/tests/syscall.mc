void exit(int retnum) {
    @(93, retnum);
}

int main() {
    char arr[6];
    arr[0] = 'h';
    arr[1] = 'e';
    arr[2] = 'l';
    arr[3] = 'l';
    arr[4] = 'o';
    arr[5] = '\n';

    @(64, 1, arr, 6);
    exit(123);
    return 0;
}
