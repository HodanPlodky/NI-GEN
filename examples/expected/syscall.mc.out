input file : examples/tests/syscall.mc
hello
return 123
